docs/quickstart.md

```diff
@@ -0,0 +1,766 @@
+---
+title: QuickStart
+description: Powertools introduction
+---
+Quickstart introducing core Powertools functionalities.
+
+
+## Installation 
+With [SAM](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) installed, you can follow this Quickstart step by step or you can build a project with the final Quickstart code you can play with.
+If you follow it all along, create a new empty project.
+=== "shell"
+```bash
+sam init --runtime python3.9 --dependency-manager pip --app-template hello-world --name powertools-quickstart
+```
+If you want to play with the final code, then you can download it from [aws-samples](https://github.com/aws-samples/cookiecutter-aws-sam-python) github repository.
+=== "shell"
+```bash
+sam init --location https://github.com/aws-samples/cookiecutter-aws-sam-python
+```
+### Configuration
+If you deploy code from Quickstart make sure to [set up your AWS credentials](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-getting-started-set-up-credentials.html).
+
+### Code example
+In the `powertools-quickstart` folder, we modify the following files:
+
+* **app.py** - Application code.
+* **template.yaml** - AWS infrastructure configuration using SAM.
+* **requirements.txt** - List of extra Python packages needed.
+
+Let's configure our base application to look like the following code snippet.
+=== "app.py"
+
+    ```python
+    def hello():
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+    def lambda_handler(event, context):
+        return hello()
+    ```
+
+=== "template.yaml"
+
+    ```yaml
+    AWSTemplateFormatVersion: '2010-09-09'
+    Transform: AWS::Serverless-2016-10-31
+    Description: >
+        Sample SAM Template for powertools-quickstart
+    Globals:
+    Function:
+        Timeout: 3
+    Resources:
+    HelloWorldFunction:
+        Type: AWS::Serverless::Function
+        Properties:
+        CodeUri: hello_world/
+        Handler: app.lambda_handler
+        Runtime: python3.9
+        Architectures:
+            - x86_64
+        Events:
+            HelloWorld:
+            Type: Api 
+            Properties:
+                Path: /hello
+                Method: get
+    Outputs:
+        HelloWorldApi:
+            Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+            Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+Our lambda code consists of the `lambda_handler` method that is invoked by the API and the `hello` method that returns the results to the API gateway and is invoked by the handler itself.
+The SAM model configures API Gateway, which redirects traffic to Lambda for one path only: `hello`.
+!!! Warning 
+    For simplicity, we do not set up authentication and authorisation in the example!
```

Comment:
Tip: Convert to an actionable suggestion in case they do want to pursue this -- As a customer, where would I go to learn more on doing this?

Reply:
Extended Warning message with link to appropiate documentation.